src/main/java/htw/kbe/maumau/domain/MauMau.java

```diff
@@ -0,0 +1,80 @@
+package htw.kbe.maumau.domain;
+
+import java.util.List;
+
+public interface MauMau {
+
+    /**
+     * Draws the given amount of cards
+     * @param amount amount of cards that will get drawn
+     * @return list of cards that will be drawn
+     */
+    List<Card> initialCardDealing(int amount);
+
+    /**
+     * shuffles the deck
+     */
+    void shuffle();
+
+    /**
+     * saves the last played card when deck is empty
+     * @return the card that was on top of the discardpile
+     */
+    Card saveLastCard();
```

Comment:
```suggestion
    Card getLastCard();
```
Die Methode speichert die letzte Karte ja nicht in dem Sinne sondern gibt sie zurück. Aber wenn wir bedenken, dass diese Methode von Deck aufgerufen wird, könnte sie sogar void sein.